File: 1672. Richest Customer Wealth/solution.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        wealthiest = 0
        for i in range(len(accounts)):
            wealth = 0
            for j in range(len(accounts[i])):
                wealth += accounts[i][j]
            if wealth > wealthiest:
                wealthiest = wealth
        return wealthiest

File: 1672. Richest Customer Wealth/solution.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let wealthiest = 0;
    for (let i = 0; i < accounts.length; i++) {
        let wealth = 0;
        for (let j = 0; j < accounts[i].length; j++) {
            wealth += accounts[i][j];
        }
        if (wealth > wealthiest) {
            wealthiest = wealth;
        }
    }
    return wealthiest;
};